src/discussions/post-comments/comments/comment/Comment.jsx

```diff
@@ -19,7 +19,12 @@ import { useUserCanAddThreadInBlackoutDate } from '../../../data/hooks';
 import { fetchThread } from '../../../posts/data/thunks';
 import LikeButton from '../../../posts/post/LikeButton';
 import { useActions } from '../../../utils';
-import { selectCommentCurrentPage, selectCommentHasMorePages, selectCommentResponses } from '../../data/selectors';
+import {
+  selectCommentCurrentPage,
+  selectCommentHasMorePages,
+  selectCommentResponses,
+  selectCommentSortOrder,
+} from '../../data/selectors';
```

Comment:
please move these selectors to one line

Reply:
Moved to multiple lines to increase readability